lark/parser_frontends.py

```diff
@@ -32,20 +32,13 @@ def __init__(self, parser_type, lexer_type):
         self.parser_type = parser_type
         self.lexer_type = lexer_type
 
-    def __call__(self, lexer_conf, parser_conf, options):
-        assert isinstance(lexer_conf, LexerConf)
-        assert isinstance(parser_conf, ParserConf)
-        parser_conf.parser_type = self.parser_type
-        lexer_conf.lexer_type = self.lexer_type
-        return ParsingFrontend(lexer_conf, parser_conf, options)
-
     def deserialize(self, data, memo, lexer_conf, callbacks, options):
         parser_conf = ParserConf.deserialize(data['parser_conf'], memo)
         parser = LALR_Parser.deserialize(data['parser'], memo, callbacks, options.debug)
         parser_conf.callbacks = callbacks
         return ParsingFrontend(lexer_conf, parser_conf, options, parser=parser)
 
-
+    # ... Continued later in the module
```

Comment:
Why?

Reply:
So it won't be part of the standalone code.

Reply:
I would suggest not splitting up the class. Instead move the complete class further down.

Reply:
It's also a bit strange to split the class in the middle. It breaks the indentation, and obfuscates the end of the standalone directive.

The only other place in Lark that happens is `tree.Tree`, and I'm not sure that it should stay that way.

But my mind isn't made up. If you think splitting the class using inheritance is too weird, I'll accept it.